collections/hrsi-psa/README.MD

```diff
@@ -0,0 +1,41 @@
+# High-Resolution Snow & Ice Monitoring; Persistent Snow Area
+
+## Short description
+
+The Persistent Snow Area (PSA) product is generated annually from FSC products and provides the extent of persistent snow cover for that year (the area where snow is present throughout the hydrological year).
+
+## Band information
+
+<table>
+  <thead>
+    <tr>
+      <th>Band Name</th>
+      <th>Sample Type</th>
+      <th>Expected Values</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>PSA </td>
+      <td >UINT 8</td>
+      <td>0 - no persistent snow, 1 - persistent snow, 255 - no data </td>
+      <td>Persistent snow area </td>
+    </tr>
+    <tr>
+      <td>QC </td>
+      <td >UINT8 </td>
+      <td>0 - high quality, 1 - medium quality, 2 - low quality, 3 - minimal quality, 255 - no data  </td>
+      <td>Persistent snow area quality layer </td>
+    </tr>         
+   </tbody>
+</table> 
+## More information  
```

Comment:
Formatting issue with 'More Information' title
<img width="220" alt="image" src="https://user-images.githubusercontent.com/49271738/225618608-7b339687-5e94-40c4-9b70-d55b086ce925.png">

after this fix , it can be merged . All good from my side

Reply:
fixed